glossary.md

```diff
@@ -0,0 +1,59 @@
+Zettle Go API Glossary
+=============
+
+<!-- Add the corresponding section if any term start with the heading letter. Add numbers before letters if there are glossary items that start with numbers.-->
+<!-- Order: numbers, then A–Z. For example, 3GPP is placed before glossary items started with letter a. --<
+<!-- Write full sentences when explaining acronyms and concepts. -->
+<!-- For syntax reference, see https://www.markdownguide.org/extended-syntax/. -->
+
+## A
+<dl>
+  <dt>Acquiring bank</dt>
+  <dd>Also known as acquirer. An acquiring bank is a financial institution that authenticates the card holder and authorises the transaction. It makes sure that the card is valid and the card has sufficient funds to complete the transaction. This process is known as clearing.</dd>
+</dl>
+
+## C
+<dl>
+  <dt>Clearing</dt>
+  <dd>After a sale is made between a merchant and a buyer, an acquiring bank authenticates the card holder and authorises the transaction. It makes sure that the card is valid and the card has sufficient funds to complete the transaction. This process is known as clearing.</dd>
+</dl>
+
+## I
+<dl>
+  <dt>Issuing bank</dt>
+  <dd>Also known as issuer. An issuing bank is a bank or financial institution that offers payment cards. For example, credit and debit cards.</dd>
+</dl>
+
+## L
+<dl>
+  <dt>Liquid account</dt>
+  <dd>A Zettle merchant account. It's used to hold funds that are ready to be paid out to merchants. After a transaction happens, the funds are recorded in the preliminary account while the acquiring bank is clearing the funds. After the funds are cleared, the funds will be moved to and recorded in the liquid account while waiting to be paid out.</dd>
+</dl>
+
+## M
+<dl>
+  <dt>Major currency unit</dt>
+  <dd>A major currency unit has a value with digits to the left of the decimal point. For example, £33.</dd>
+  <dt>Minor currency unit</dt>
+  <dd>A minor currency unit has a value that is 1/100 or less than the major currency unit. Only two-decimal currencies like US dollars have a minor currency unit. For example, $1.99. Zero-decimal currencies only have a major currency unit like Japanese Yen. For example, ¥1095.</dd>
+</dl>
+
+## P
+<dl>
+  <dt>Preliminary account</dt>
+  <dd>A Zettle merchant account. It's used to hold funds that are being cleared by the issuing bank. After a transaction happens, the funds are recorded in the preliminary account while the acquiring bank is clearing the funds. After the funds are cleared, the funds will be moved to and recorded in the liquid account while waiting to be paid out.</dd>
```

Comment:
Same thing as the liquid account. We should not mention clearing with acquiring banks here as it is not true.